### ISSUE TITLE
[a11y] Issue 39: Opinion Group chart lacks semantic descriptions

### ISSUE THREAD

**Issue by kk-slider18** [NewJerseyStyle/polis#135]
**Description:**
Each group in the Opinion Group chart lacks semantic descriptions, making it difficult for screen reader users to understand the data. Proper ARIA attributes or alternative text should be added.

**Component/Area:** Open Participation page

**Source:** Screen Reader

**Affected Elements:** ARIA attributes, alt text

**Comment by YumiChen** [comment]
Just a thought, there was a related [a11y talk](https://www.linkedin.com/posts/ugcPost-7303587241744863232-dhHr), according to which probably we can utilize both:
- alternative texts for the chart
- hidden data tables

The chart is currently not able to be navigated with keyboard, either. Probably we also want to define what to be announced when a group in the chart is focused
